### DISCUSSION TITLE
Error response when DPoP token is used as bearer token

### DISCUSSION THREAD

**Discussion by ashwin-yo** [ietf-tools/datatracker#5147] [Help / Questions] [answered]
For the [OAuth DPoP](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-dpop), what should the error response be when a DPoP token is used as a bearer token?

For example:
```
GET /protected-resource HTTP/1.1
Host: resource.example.org
Authorization: Bearer VALID_DPoP_TOKEN
```

Should the error code be 400 or 401, and what should be the error and error_description here?

**Comment by rjsparks** [comment, marked as answer]
This isn't the right forum for that question (this is for datatracker questions). - you need to be asking the OAuth WG. Email to oauth@ietf.org is your best option.

**Reply by ashwin-yo** [reply]
Thank you, I have sent the email.
